src/main/kotlin/com/openlattice/data/DataGraphService.kt

```diff
@@ -615,4 +624,48 @@ open class DataGraphService(
 //        }.stream()
         return Stream.empty()
     }
+
+    override fun getExpiringEntitiesFromEntitySet(entitySetId: UUID, expirationPolicy: DataExpiration,
+                                                  dateTime: OffsetDateTime, deleteType: DeleteType,
+                                                  expirationPT: Optional<PropertyType>): BasePostgresIterable<UUID> {
```

Comment:
either this `expirationPT` or `expirationPropertyType` in `DataGraphManager` should be renamed so they are both the same